src/components/Trade/TradeTabs/Ranges/Leaderboard.tsx

```diff
@@ -255,11 +222,41 @@ function Leaderboard() {
         />
     ));
 
+    // TODO: we can probably severely reduce the number of wrappers in this JSX
+
     return (
         <FlexContainer flexDirection='column' fullHeight>
-            <div>{headerColumnsDisplay}</div>
+            <RangeRowStyled size={tableView} leaderboard header>
+                {headerColumns.map((header, idx) => (
+                    <RangeHeader
+                        key={idx}
+                        sortBy={sortBy}
+                        setSortBy={setSortBy}
+                        reverseSort={reverseSort}
+                        setReverseSort={setReverseSort}
+                        header={header}
+                    />
+                ))}
+            </RangeRowStyled>
             <div style={{ flex: 1, overflow: 'auto' }}>{rowItemContent}</div>
-            <div>{footerDisplay}</div>
+            <FlexContainer
+                as='footer'
+                alignItems='center'
+                justifyContent='center'
+                gap={isSmallScreen ? 4 : 8}
+                margin='0 auto'
```

Comment:
This line is the actual substantive change. All other changes are code cleanup to better organize the file and create fewer unnecessary wrappers in the DOM.